docs/concepts/12-fees.mdx

```diff
@@ -81,56 +81,94 @@ the two fees.
 
 :::
 
-### MerkleFactory
-
-The ONLY contracts that can charge fees in Sablier are the `MerkleFactory` contract and the airdrop contracts it
-deploys.
-
-The [Protocol Admin](/concepts/governance/) can update the fee in the Merkle Factories contract. However, the new fee
-applies only to airdrops created after the update. The fee values in the individual airdrop contracts remain immutable
```

Comment:
The immutability is not true but the fee in campaign can only go down. So should we mention that?